rpc/backend/utils.go

```diff
@@ -118,16 +118,18 @@ func (b *Backend) getAccountNonce(accAddr common.Address, pending bool, height i
 }
 
 // CalcBaseFee calculates the basefee of the header.
-func CalcBaseFee(config *params.ChainConfig, parent *ethtypes.Header, p feemarkettypes.Params) *big.Int {
+func CalcBaseFee(config *params.ChainConfig, parent *ethtypes.Header, p feemarkettypes.Params) (*big.Int, error) {
 	// If the current block is the first EIP-1559 block, return the InitialBaseFee.
 	if !config.IsLondon(parent.Number) {
-		return new(big.Int).SetUint64(params.InitialBaseFee)
+		return new(big.Int).SetUint64(params.InitialBaseFee), nil
+	}
+	if p.ElasticityMultiplier == 0 {
```

Comment:
```
        // CONTRACT: ElasticityMultiplier cannot be 0 as it's checked in the params
	// validation
```

I think we just need to put this check into the `Validate` method of the params.